boards/arm/stm32h573i_dk/stm32h573i_dk.dts

```diff
@@ -16,7 +16,7 @@
 	chosen {
 		zephyr,console = &usart1;
 		zephyr,shell-uart = &usart1;
-		zephyr,sram = &sram0;
+		zephyr,sram = &sram1;
```

Comment:
Is this related to adding ethernet?

Reply:
I complied with the reference manual RM0481.pdf
see:
2.3.2 Memory map and register boundary addresses: only sram1,sram2,sram3 exist.

the linker use sram3 in soc/arm/st_stm32/stm32h5/sections.ld
 see  : add Ethernet driver for stm32 h5xx #59613